DEPRECATIONS-v2.md

```diff
@@ -0,0 +1,21 @@
+# Deprecations
+
+Introducing deprecations and feature flags to be removed in the version 2 lists.
+
+💁 Please follow these steps to safely upgrade your Design System to Spirit Design System v2 components.
+
+## spirit-web v2
```

Comment:
To be honest, the different version numbers seem weird to me…

> Spirit Design System v2 components

The problem is we technically don't have a single version of the design system as a whole.

I think we should either:

- align the package version numbers after all so we do have the same version everywhere, or
- put only the latest deprecations here and call it simply `DEPRECATIONS.md`. Older deprecation versions would remain in the packages themselves.

Reply:
I think that aligning all the package versions will never happen. And if so then unintentionally. It was my suggestion to create a `DEPRECATION.md` in the root of the monorepo as a signpost to other deprecations files in the packages. Since we know that these files will be created and removed in favor of `UPGRADE/MIGRATION` files, maybe it is a bigger topic to discuss how we should handle them.

Maybe there should be only one `MIGRATION.md` file in the root which will point both to the package upgrade file or deprecation file 🤷